packages/esm-ward-app/src/types/index.ts

```diff
@@ -50,3 +50,12 @@ interface BedTagMap {
 }
 
 export type BedStatus = 'AVAILABLE' | 'OCCUPIED';
+
+export interface InstalledBackendModules {
```

Comment:
I wonder if it would be better to name this "SystemInformation" or something more generic.